panopticum/management/commands/config/default.json

```diff
@@ -0,0 +1,305 @@
+      "iOS Device":{
+         "order":8,
+         "description":"iOS device"
+      },
+      "Android Device":{
+         "order":9,
+         "description":"Android device"
+      },
+      "VMWare Appliance":{
+         "order":11,
+         "description":"The Virtual Appliance Marketplace contains a variety of virtual appliances packaged in OVF format that you can download and deploy in your vSphere environment."
+      },
+      "TBD":{
+         "order":12,
+         "description":"To be filled later by user."
+      },
+      "OSA":{
+         "order":13,
+         "description":"OpenStack-Ansible (OSA) uses the Ansible IT automation engine to deploy an OpenStack environment on Ubuntu Linux. https://docs.openstack.org/project-deploy-guide/openstack-ansible/ocata/overview-osa.html"
+      }
+   },
+   "DeploymentLocationMapping": {
+      "cloud-only": "Cloud datacenter",
+      "cloud": "Cloud datacenter",
+      "PDS operator side": "On-prem centralized",
+      "on-prem": "On-prem centralized",
+      "on-prem-only": "On-prem centralized",
+      "azure": "Microsoft Azure",
+      "aws": "Amazon Web Services",
+      "plesk": "Plesk",
+      "ios": "iOS Device",
+      "android": "Android Device",
+      "vmware appliance": "VMWare Appliance",
+      "Agent": "On-prem agent unit",
+      "osa": "OSA",
+      "TBD": "TBD"
+   },
+   "ProgrammingLanguages":[
+      "Go-lang",
+      "Python",
+      "C/C++",
+      "Javascript",
+      "Java",
+      "Ruby",
+      "Erlang",
+      "shellscript",
+      "Perl",
+      "Objective-C",
+      "Android",
+      "Powershell-script",
+      ".Net",
+      "PHP"
+   ],
+   "ApplicationVendors":[
+      "Acronis",
+      "OpenSource",
+      "Percona",
+      "Google",
+      "Commercial"
+   ],
+   "ApplicationFrameworks":{
+      "tornado":"Python",
+      "sqlalchmey":"Python",
+      "fastcgi":"Python",
+      "storedprocs":"Python",
+      "aiohttp":"Python",
+      "angular4":"Javascript",
+      "vuejs":"Javascript",
+      "angular1":"Javascript",
+      "laravel":"PHP",
+      "werkzeug":"Python",
+      "flask":"Python",
+      "go-std":"Go-lang",
+      "beego":"Go-lang",
+      "xorm":"Go-lang",
+      "gorm":"Go-lang",
+      "WPF":".Net",
+      "angular2":"Javascript",
+      "extjs4":"Javascript",
+      "gevent":"Python",
+      "ruby on rails":"Ruby",
+      "android sdk":"Android",
+      "wix":"C/C++",
+      "phantomjs":"Python",
+      "asyncio":"Python",
+      "go-micro":"Go-lang",
+      "gin":"Go-lang",
+      "gorp":"Go-lang",
+      "DBIx":"Perl",
+      "Mason":"Perl",
+      "boost":"C/C++"
+   },
+   "ProductVersions":{
+      "Company Cloud 1.0":{
+         "shortname":"ComCloud1.0",
+         "order":1,
+         "family":"Cloud Services"
+      },
+      "Google Maps 1.0":{
+         "shortname":"Maps1.0",
+         "order":2,
+         "family":"Cloud Services"
+      }
+   },
+   "ProductFamilies":[
+      "Cloud Services",
+      "Native Applications"

```

Comment:
"Cloud Services" and "Native Applications" sound like a 'component type'. ProductFamilies should be rather "My Cloud Product for Business", "My On-prem Product for Business", "My Desktop Product for Consumers"

https://github.com/perfguru87/panopticum/raw/master/panopticum/static/images/panopticum-model.png

Reply:
Sure will align to panopticum-model.png